icepyx/tests/test_auth.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import requests
+
```

Comment:
should we combine this suite with the `test_Earthdata.py`? The earthdata tests are essentially checking the login methods, which are conceptually part of auth as well, though also not technically directly testing pieces of auth. Open to thoughts and suggestions on what makes the most sense.

Reply:
> I have a very slight lean towards keeping them separate,

A slight lean is enough argument for me!